README.md

```diff
@@ -155,7 +155,7 @@ merged, we need to test them on all supported distributions and
 architectures.
 
 When you submit patches to GitHub, oVirt's Jenkins CI will run its tests
-according to configuration in the stdci.yaml file.
+according to configuration in the `stdci.yaml` file.
```

Comment:
We don't use stdci.yml since we moved to github. This section need to be rewritten instead beautifying stale text.